tests/sles4sap/migrate_sles_to_sles4sap.pm

```diff
@@ -36,6 +36,7 @@ sub run {
     wait_serial "Please enter your activation code", timeout => 5;
     type_string "${regcode}\n";
     assert_script_run "ls /tmp/OK" || die "Migration failed!";
+    zypper_call "in -y -t pattern sap_server";
```

Comment:
I'm not sure if `-l` (auto-accept license) option is not needed also with `-y`.

Reply:
`-l` is not needed.